site/blog/starlingx-release-2-security-features.md

```diff
@@ -0,0 +1,23 @@
+---
+title:  Securing the Edge with StarlingX
+author: ian-jolliffe
+date: 2019-10-30T01:32:05.627Z
+category: features-and-updates
+---
+Security on the edge is crucial. Learn more about how the StarlingX 2.0 release helps addressing this challenge. <!-- more -->
+
+
+As we drive compute to the edge of the network many things change and become more interesting and new problems need to be solved. At a minimum the challenges include space and power limitations which then turns into balancing server costs while enabling emerging applications and new transformational use cases at the edge when you take the next steps. However; even though the way compute is delivered at the edge is different, the need for security remains constant or even more critical.
```

Comment:
At a minimum, the challenges include space and power limitations.  The next level of challenge includes balancing server costs while enabling emerging applications and new transformational use cases at the edge when you take the next steps.

Reply:
I added the changes to eliminate using 'challenge include' twice and the 'next level of challenge' sounded a bit odd to me.